core/chains/evm/gas/cmd/arbgas/main.go

```diff
@@ -9,22 +9,22 @@ import (
 
 	"github.com/ethereum/go-ethereum/ethclient"
 	"github.com/ethereum/go-ethereum/rpc"
-	"go.uber.org/zap/zapcore"
 
+	"github.com/smartcontractkit/chainlink-common/pkg/logger"
 	feetypes "github.com/smartcontractkit/chainlink/v2/common/fee/types"
 	"github.com/smartcontractkit/chainlink/v2/core/chains/evm/assets"
 	"github.com/smartcontractkit/chainlink/v2/core/chains/evm/gas"
-	"github.com/smartcontractkit/chainlink/v2/core/logger"
 )
 
 func main() {
 	if l := len(os.Args); l != 2 {
 		log.Fatal("Expected one URL argument but got", l-1)
 	}
 	url := os.Args[1]
-	lggr, sync := logger.NewLogger()
-	defer func() { _ = sync() }()
-	lggr.SetLogLevel(zapcore.DebugLevel)
+	lggr, err := logger.New()
+	if err != nil {
+		panic(err)
```

Comment:
```suggestion
		log.Fatal("Failed to create logger:", err)
```

Reply:
I think we should exit if we have no logger, so perhaps:
```suggestion
		log.Fatal("Failed to create logger:", err)
		return
```

Reply:
log.Fatal already exits:
https://cs.opensource.google/go/go/+/refs/tags/go1.21.4:src/log/log.go;l=411-413